nextflow.config

```
@@ -6,27 +6,70 @@
 ----------------------------------------------------------------------------------------
 */
 
+
 // Global default params, used in configs
 params {
 
-    // TODO nf-core: Specify your pipeline's command line flags
     // Input options
-    input                      = null
+    input = null
+    input_basedir = null
+
+    // Preqc options
+    trim_front = 0
+    trim_tail = 0
+    max_length = null
+    min_length = null
+    max_n_bases = 0
+    avg_qual = 20
+    dedup = false
+    remove_polyg = false
+
+    // adapterqc options
+    adapterqc_mismatches = 0.1
+
+    // demux options
+    demux_mismatches = 0.1
+    demux_min_length = null
+
+    // collapse options
+    markers_ignore = null
+    algorithm = 'adjacency'
+    max_neighbours = 60
+    collapse_mismatches = 2
+    collapse_min_count = 2
+    collapse_use_counts = false
+
+    // graph options
+    multiplet_recovery = true
+    leiden_iterations = 10
+    graph_min_count = 2
+
+    // annotate options
+    min_size = null
+    max_size = null
+    dynamic_filter = 'min'
+    aggregate_calling = true
+
+    // analysis options
+    compute_polarization = true
+    compute_colocalization = true
+    use_full_bipartite = false
+    polarization_normalization = "clr"
+    polarization_binarization = false
+    colocalization_transformation = "log1p"
+    colocalization_neighbourhood_size = 1
+    colocalization_n_permutations = 50
+    colocalization_min_region_count = 5
 
+    // skip options
+    skip_report   = false
+    skip_analysis = false
 
-    // References
-    genome                     = null
-    igenomes_base              = 's3://ngi-igenomes/igenomes'
-    igenomes_ignore            = false
-    // MultiQC options
-    multiqc_config             = null
-    multiqc_title              = null
-    multiqc_logo               = null
-    max_multiqc_email_size     = '25.MB'
-    multiqc_methods_description = null
+    // Main pixelator container override
+    pixelator_container        = null
 
     // Boilerplate options
-    outdir                     = null
+    outdir                     = "./results"
```

```suggestion
    outdir                     = null
```

on s3 we need to overwrite that by default, so we usually keep null by default